Assets/MRTK/Providers/WindowsMixedReality/XR2018/Controllers/HPMotionController.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Unity.Profiling;
+using UnityEngine;
+using UnityEngine.XR;
+using System;
+using UnityEngine.XR.WSA.Input;
+
+#if HP_CONTROLLER_ENABLED
+using Microsoft.MixedReality.Input;
+using MotionControllerHandedness = Microsoft.MixedReality.Input.Handedness;
+using Handedness = Microsoft.MixedReality.Toolkit.Utilities.Handedness;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input
+{
+    [MixedRealityController(
+        SupportedControllerType.HPMotionController,
+        new[] { Handedness.Left, Handedness.Right })]
+    public class HPMotionController : WindowsMixedRealityController
+    {
+#if HP_CONTROLLER_ENABLED
+        internal HPMotionControllerInputHandler inputHandler;
+        internal MotionControllerState MotionControllerState;
```

Comment:
Inconsistent capitalization on these fields. Could they be turned into auto-properties and both in PascalCase?